src/modules/75-ci/components/PipelineStudio/BuildInfraSpecifications/BuildInfraSpecifications.tsx

```diff
@@ -602,12 +628,135 @@ export default function BuildInfraSpecifications({ children }: React.PropsWithCh
             disableTypeSelection: true
           }}
           disabled={isReadonly}
+          restrictToSingleEntry={restrictToSingleEntry}
         />
       </Container>
     ),
     []
   )
 
+  const renderContainerSecurityContext = React.useCallback(({ formik }: { formik: any }): JSX.Element => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232285)
```suggestion
  const renderContainerSecurityContext = React.useCallback(({ formik }: { formik: unknown }): JSX.Element => {
```